.github/workflows/main.yml

```diff
@@ -19,3 +19,16 @@ jobs:
       - uses: ./
         with:
           token: ${{ secrets.TOKEN }}
+      - name: switching from HTTPS to SSH
+        run: git remote set-url origin ${{ secrets.SSH }}
```

Comment:
hm, what's the explanation here? and why is this a secret?

Reply:
It's supposed to be this: `git@github.com:ljharb/repo-report-cache.git`
But, I do not see the point in keeping it a `secret`. Am updating it in the next commit

Reply:
does it not work if it's https?